src/assets/Generator.Shared/Utf8JsonWriterExtensions.cs

```diff
@@ -140,5 +141,53 @@ public static void WriteObjectValue(this Utf8JsonWriter writer, object value)
                     throw new NotSupportedException("Not supported type " + value.GetType());
             }
         }
+
+        public static void WriteBinaryDataValue(this Utf8JsonWriter writer, BinaryData value)
```

Comment:
It would be good to get the Azure.Core v-team's eyes on this change per the change to Azure.Core shared source.  @jsquire, @JoshLove-msft, @christothes